### PR TITLE
Updates Italian translations

### PR DESCRIPTION
**Pull request by Dragone2**
Hi,
I've reviewed some of the Italian translations.
Have a nice day :)

### CODE REVIEW

**Comment by SadieCat** [comment]
Did you run `./language/update.sh` (only commit the changes your language file) before updating? I don't think the language files are up to date as `POT-Creation-Date` hasn't changed.

**Comment by Dragone2** [comment]
Hi,
I forgot to run `./language/update.sh` and now it's ok. Thanks :)

File: language/anope.it_IT.po [hunk 1]
```diff
@@ -9359,3 +9084,6 @@ msgstr "{nick | canale}"
 
 msgid "{nick | channel} memo-text"
 msgstr "{nick | canale} testo-memo"
+
+#~ msgid "Password authentication required for that command."
+#~ msgstr "Autenticazione con password richiesta per questo comando."
```
Comment (SadieCat) [inline language/anope.it_IT.po:9089]:
```suggestion
```

These are unused now and can be removed.

**Comment by SadieCat** [comment]
Merged, thanks for your contribution! 👍

**Pull request merged**